me.simakis.dropboxignore.metainfo.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
I think this may actually want to declare itself as a console-applicatio rather than desktop-application. https://www.freedesktop.org/software/appstream/docs/sect-Metadata-ConsoleApplication.html there might be a little knock on from that but this will allow software stores to present it more accurately to users.